src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/ElementUtils.kt

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.performance.tools.jiraactions.api.webdriver
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.Keys
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+class ElementUtils {
+    companion object {
+        @JvmStatic
+        fun sendKeysWhenClickable(driver: WebDriver, locator: By, vararg keysToSend: CharSequence): WebElement {
+            val findElement = driver.findElement(locator)
+            return sendKeysWhenClickable(driver, findElement, *keysToSend)
+        }
+
+        fun sendKeysWhenClickable(driver: WebDriver, element: WebElement, vararg keysToSend: CharSequence): WebElement {
+            driver.wait(ExpectedConditions.elementToBeClickable(element))
+            element.click()
+
+            element.sendKeys(*keysToSend)
+            return element
+        }
+
+        @JvmStatic
+        fun sendKeysAndValidate(driver: WebDriver, locator: By, text: String): WebElement {
+            val element = sendKeysWhenClickable(driver, locator, text as CharSequence)
+
+            // It's hard to say when the keys can be sent. They seem to randomly get lost.
```

Comment:
Truth be told, sometimes I encounter https://ecosystem.atlassian.net/browse/JPERF-298 as a real user.
When I tab and type too quickly, beginning of my inputs get dropped.
Such are our frontends right now.

Reply:
👍